engine/framework/base_jobmaster.go

```diff
@@ -226,20 +227,22 @@ func (d *DefaultBaseJobMaster) Close(ctx context.Context) error {
 
 	d.master.doClose()
 	d.worker.doClose()
```

Comment:
Only for discussion. 
Things become complicated for the cleaning of framework resource.
When we talk about `doClose` before, we may refer to the `release the framework resource`.

Reply:
If we need to distinguish some detail `Close` method, may be a more specified name is better, like `doCloseWithoutMessageHandler`.